apps/design/frontend/src/system_settings_screen.tsx

```diff
@@ -228,22 +315,211 @@ export function SystemSettingsForm({
+                        auth: {
+                          ...systemSettings.auth,
+                          arePollWorkerCardPinsEnabled: isChecked,
+                        },
+                      })
+                    }
+                    disabled={!isEditing}
+                  />
+                  <InputGroup label="Inactive Session Time Limit">
+                    <SearchSelect
+                      value={systemSettings.auth.inactiveSessionTimeLimitMinutes.toString()}
+                      options={[
+                        { value: '10', label: '10 minutes' },
+                        { value: '15', label: '15 minutes' },
+                        { value: '20', label: '20 minutes' },
+                        { value: '30', label: '30 minutes' },
+                        { value: '360', label: '6 hours' },
+                      ]}
+                      onChange={(newValue) => {
+                        setSystemSettings({
+                          ...systemSettings,
+                          auth: {
+                            ...systemSettings.auth,
+                            inactiveSessionTimeLimitMinutes:
+                              safeParseInactiveSessionTimeLimit(newValue),
+                          },
+                        });
+                      }}
+                    />
+                  </InputGroup>
+                  <InputGroup label="Incorrect Pin Attempts Before Lockout">
+                    {/* <input
+                      value={
+                        systemSettings.auth
+                          .numIncorrectPinAttemptsAllowedBeforeCardLockout
+                      }
+                      type="number"
+                      step={1}
+                      min={3}
+                      max={10}
+                      onChange={(e) => {
+                        setSystemSettings({
+                          ...systemSettings,
+                          auth: {
+                            ...systemSettings.auth,
+                            numIncorrectPinAttemptsAllowedBeforeCardLockout:
+                              safeParseIncorrectPinAttempts(
+                                e.target.valueAsNumber
+                              ),
+                          },
+                        });
+                      }}
+                    /> */}
+                    <SearchSelect
+                      value={systemSettings.auth.numIncorrectPinAttemptsAllowedBeforeCardLockout.toString()}
+                      options={[
+                        { value: '3', label: '3' },
+                        { value: '4', label: '4' },
+                        { value: '5', label: '5' },
+                        { value: '6', label: '6' },
+                        { value: '7', label: '7' },
+                        { value: '8', label: '8' },
+                        { value: '9', label: '9' },
+                        { value: '10', label: '10' },
+                      ]}
```

Comment:
I made this a dropdown because

1. 3-10 might be an unintuitive range for the user, and this makes it more clear than a number spinner what the options are
2. It avoids an annoying edge case where typing `10` is impossible because validation happens immediately on change and `1` is an invalid option. This can be worked around but the dropdown felt simpler.